src/ApiTestCase.php

```diff
@@ -228,37 +214,100 @@ private function getCalledClassFolder()
         $calledClass =  get_called_class();
         $calledClassFolder = dirname((new \ReflectionClass($calledClass))->getFileName());
 
-        $this->folderExists($calledClassFolder);
+        $this->directoryCheck($calledClassFolder);
 
         return $calledClassFolder;
     }
 
+    protected function loadAllFixtures()
+    {
+        $this->loadAllFixtures();
```

Comment:
I know that nothing special happening here but i thought it increase developer experiance.